### ISSUE TITLE
fix all the warnings/errors

### ISSUE THREAD

**Issue by janzandr** [EnMAP-Box/enmap-box-documentation#10]
Hi Fabian,
not sure if all those warnings are introduced by the repo-restructering on GitHub, but it would be really nice, if you could have a look at it. 

```
PC-21-0602+Andreas@PC-21-0602 MINGW64 /d/source/QGISPlugIns/enmap-box-documentation (main)
$ ./make.bat html
Sphinx v5.0.0 in Verwendung
making output directory... erledigt
loading intersphinx inventory from https://docs.python.org/objects.inv...
intersphinx inventory has moved: https://docs.python.org/objects.inv -> https://docs.python.org/3/objects.inv
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 250 source files that are out of date             
updating environment: [new config] 250 added, 0 changed, 0 removed             
reading sources... [100%] usr_section/workshop_tutorials/index                 rt1tion/random_points_from_mask_raster_layerr_layer_layer_wavelength_and_fwhm_curvesraster_
WARNING: autodoc: failed to import module 'algorithmprovider' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'                                                                                               
WARNING: autodoc: failed to import module 'dependencycheck' from module 'enmapbox'; the following exception was raised:  
No module named 'enmapbox'                                                                                               
WARNING: autodoc: failed to import module 'enmapboxplugin' from module 'enmapbox'; the following exception was raised:   
No module named 'enmapbox'                                                                                               
WARNING: autodoc: failed to import module 'resources' from module 'enmapbox'; the following exception was raised:        
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'testing' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.about' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.applications' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.datasourcemanager' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.datasources' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.dockmanager' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.docks' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.enmapboxgui' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.mapcanvas' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.mimedata' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.settings' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.utils' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.ui' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.widgets.drawing' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.widgets.models' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.widgets.rasterbandselectionwidget' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.widgets.trees' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
WARNING: autodoc: failed to import module 'gui.widgets' from module 'enmapbox'; the following exception was raised:
No module named 'enmapbox'
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\api\hubdc.rst:4: WARNING: toctree contains reference to nonexisting document 'dev_section/dev_cookbook/hubdc/source/api/hubdc_core'
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\api\hubdc.rst:4: WARNING: toctree contains reference to nonexisting document 'dev_section/dev_cookbook/hubdc/source/api/hubdc_applier'
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_enmapboxrepository.rst:20: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_enmapboxrepository.rst:58: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_enmapboxrepository.rst:58: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_enmapboxrepository.rst:58: WARNING: Inline emphasis start-string without end-string.
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_installation.rst:8: WARNING: Duplicate explicit target name: "conda".
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_installation.rst:8: WARNING: Duplicate explicit target name: "conda".
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\dev_installation.rst:229: ERROR: Duplicate target name, cannot be used as a unique reference: "qgis".
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_setup\ps.rst:: WARNING: image file not readable: dev_section/programming_tutorials/programming_setup/content_root.png
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:14: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:14: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
source/dev_section/programming_tutorials/programming_tutorial2/part1_basics.rst:6: WARNING: duplicate label progtt_basics, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tutorial2\part1_basics.
rst
source/dev_section/programming_tutorials/programming_tutorial2/part1_basics.rst:12: WARNING: duplicate label progtt_basics_exercise1_hello_world, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_
tutorial2\part1_basics.rst
source/dev_section/programming_tutorials/programming_tutorial2/part1_basics.rst:96: WARNING: duplicate label basics_exercise1_hello_real_world, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tu
torial2\part1_basics.rst
source/dev_section/programming_tutorials/programming_tutorial2/part1_basics.rst:221: WARNING: duplicate label basics_exercise3_create_widgets, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tut
orial2\part1_basics.rst
source/dev_section/programming_tutorials/programming_tutorial2/part3_advanced.rst:6: WARNING: duplicate label progtt_advanced, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tutorial2\part3_adv
anced.rst
source/dev_section/programming_tutorials/programming_tutorial2/part3_advanced.rst:11: WARNING: duplicate label classification_schemes, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tutorial2\p
art3_advanced.rst
source/dev_section/programming_tutorials/programming_tutorial2/part3_advanced.rst:228: WARNING: duplicate label iconselect_tool, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tutorial2\part3_a
dvanced.rst
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\rfc_list\rfc0002.rst:35: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\rfc_list\rfc0003.rst:94: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\general\contribute.rst:4: WARNING: Duplicate explicit target name: "pull requests".
D:\source\QGISPlugIns\enmap-box-documentation\source\general\contribute.rst:212: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\general\contribute.rst:4: WARNING: Duplicate explicit target name: "here".
D:\source\QGISPlugIns\enmap-box-documentation\source\general\contribute.rst:311: WARNING: Explicit markup ends without a blank line; unexpected unindent.
source/dev_section/external_links.rst:1101: WARNING: Duplicate explicit target name: "qgis".
D:\source\QGISPlugIns\enmap-box-documentation\source\general\faq_trouble.rst:13: WARNING: Duplicate explicit target name: "here".
D:\source\QGISPlugIns\enmap-box-documentation\source\general\faq_trouble.rst:177: WARNING: Enumerated list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\general\faq_trouble.rst:207: WARNING: Enumerated list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\general\glossary.rst:87: WARNING: glossary terms must not be separated by empty lines
D:\source\QGISPlugIns\enmap-box-documentation\source\general\styleguide.rst:3: WARNING: Title overline too short.

=====
Documentation Guidelines
=====
D:\source\QGISPlugIns\enmap-box-documentation\source\index.rst:44: WARNING: Title underline too short.

EnMAP-Box Version 3.10 released
==============================
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\biomass_regression\tutorial.rst:155: WARNING: Bullet list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\biomass_regression\tutorial.rst:431: WARNING: image file not readable: usr_section/application_tutorials/biomass_regression/img/icons/numpy.png
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\biomass_regression\tutorial.rst:431: WARNING: image file not readable: usr_section/application_tutorials/biomass_regression/img/icons/numpy.png
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\biomass_regression\tutorial.rst:75: WARNING: download file not readable: D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\biomass_regressi
on\You can download the data for this exercise here:
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\engeomap\tutorial_engeomap.rst:36: WARNING: Title underline too short.

1.   Data Preparation and Requirements
-------------------------------------
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\engeomap\tutorial_engeomap.rst:36: WARNING: Title underline too short.

1.   Data Preparation and Requirements
-------------------------------------
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\urban_unmixing\tutorial.rst:81: WARNING: download file not readable: D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\application_tutorials\urban_unmixing\You c
an download the data here:
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_gettingstarted.rst:135: WARNING: Explicit markup ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_gettingstarted.rst:139: WARNING: Inline strong start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_gettingstarted.rst:147: WARNING: Explicit markup ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_gettingstarted.rst:186: ERROR: Unknown directive type "figrue".

.. figrue:: ../img/AddProfiles.png
   :width: 100%
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_installation.rst:6: WARNING: Duplicate explicit target name: "here".
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_installation.rst:191: WARNING: Enumerated list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_installation.rst:6: WARNING: Duplicate explicit target name: "here".
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_installation.rst:84: ERROR: Unknown target name: "numpy <https://numpy.org/ >".
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_installation.rst:85: ERROR: Unknown target name: "scipy <https://scipy.org >".
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\applications.rst:55: WARNING: Bullet list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\classification\classification_workflow.rst:5: WARNING: duplicate label classification workflow, other instance in D:\source\QGISPlugIns\enmap-box-documentation\sourc
e\usr_section\usr_manual\applications.rst
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_gaussian_gradient_magnitude_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.    
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_generic_filter.rst:30: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_laplace_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_maximum_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_median_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_minimum_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_binary_closing_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.   
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_binary_dilation_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.  
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_binary_erosion_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.   
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_binary_fill_holes_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.

D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_binary_opening_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.   
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_black_top-hat_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.    
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_gradient_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_grey_dilation_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.    
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_grey_erosion_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.     
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_grey_opening_filter.rst:26: WARNING: Definition list ends without a blank line; unexpected unindent.     
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_laplace_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_morphological_white_top-hat_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.    
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_percentile_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_prewitt_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\convolution__morphology_and_filtering\spatial_sobel_filter.rst:24: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\debugging\index.rst:6: WARNING: toctree glob pattern '*' didn't match any documents
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\import_data\import_landsat_l2_product.rst:13: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\masking\create_mask_raster_layer.rst:39: WARNING: Definition list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\raster_conversion\translate_raster_layer.rst:43: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\raster_conversion\write_envi_header.rst:7: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\regression\regression_workflow.rst:5: WARNING: duplicate label regression workflow, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\usr_sectio
n\usr_manual\applications.rst
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_custom_sensor_.rst:42: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_desis_hsi_.rst:31: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_enmap_.rst:31: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_landsat_4_5_tm_.rst:34: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_landsat_7_etm+_.rst:34: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_landsat_8_9_oli_.rst:35: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_prisma_.rst:31: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_sentinel-2_msi_.rst:41: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_algorithms\spectral_resampling\spectral_resampling__to_spectral_raster_layer_wavelength_and_fwhm_.rst:21: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_datatypes.rst:25: WARNING: Bullet list ends without a blank line; unexpected unindent.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\spectral_library.rst:73: ERROR: Error in "csv-table" directive:
invalid option value: (option: "width"; value: 'auto')
not a positive measure of one of the following units:
"em" "ex" "px" "in" "cm" "mm" "pt" "pc" "%".

.. csv-table::
   :header-rows: 1
   :align: center
   :width: auto

   Button, Description
   |plus_green|,  add a new profile source entry
   |cross_red|, remove selected entries
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\spectral_library.rst:473: WARNING: Inline interpreted text or phrase reference start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\spectral_library.rst:484: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\spectral_library.rst:484: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\spectral_library.rst:495: WARNING: Inline emphasis start-string without end-string.
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\spectral_library.rst:53: WARNING: duplicate label spectral_profile_sources, other instance in D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\gui.rst
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\workshop_tutorials\earsel2022\tutorial_part1.rst:33: WARNING: download file not readable: D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\workshop_tutorials\earsel2022\You can downl
oad the data for this exercise here:
looking for now-outdated files... none found
pickling environment... erledigt
checking consistency... D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\api\modules.rst: WARNING: document isn't included in any toctree
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\index.rst: WARNING: document isn't included in any toctree
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_setup\ps.rst: WARNING: document isn't included in any toctree
D:\source\QGISPlugIns\enmap-box-documentation\source\dev_section\programming_tutorials\programming_tutorial2\setup_pycharmproject.rst: WARNING: document isn't included in any toctree
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_datatypes.rst: WARNING: document isn't included in any toctree
erledigt
preparing documents... erledigt
writing output... [100%] usr_section/workshop_tutorials/index                  t1tion/random_points_from_mask_raster_layerr_layer_layer_wavelength_and_fwhm_curvesraster_
source/icon_links.rst:129: WARNING: term not in glossary: fields
source/icon_links.rst:129: WARNING: term not in glossary: continuous-valued variables
D:\source\QGISPlugIns\enmap-box-documentation\source\general\glossary.rst:172: WARNING: term not in glossary: targets
D:\source\QGISPlugIns\enmap-box-documentation\source\general\glossary.rst:178: WARNING: term not in glossary: validation set
D:\source\QGISPlugIns\enmap-box-documentation\source\general\glossary.rst:222: WARNING: term not in glossary: features
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_cookbook\classification.rst:25: WARNING: undefined label: 
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_cookbook\filtering.rst:13: WARNING: undefined label: testdataset
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_cookbook\filtering.rst:31: WARNING: undefined label: testdataset
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_cookbook\generic_filter.rst:37: WARNING: unknown keyword: generic_filter
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_gettingstarted.rst:28: WARNING: undefined label: processing algorithms
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_gettingstarted.rst:41: WARNING: undefined label: processing algorithms
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_datatypes.rst:32: WARNING: undefined label: shp_berlingradient
D:\source\QGISPlugIns\enmap-box-documentation\source\usr_section\usr_manual\processing_datatypes.rst:200: WARNING: Pygments Lexer Name 'csv' ist unbekannt
generating indices... genindex erledigt
writing additional pages... search erledigt
copying images... [100%] usr_section/workshop_tutorials/earsel2022/img/dataset.png17_accuracies.pngpngggling_new.pngh_mapcanvas.pngles.png
copying static files... erledigt
copying extra files... erledigt
dumping search index in English (code: en)... erledigt
dumping object inventory... erledigt
build abgeschlossen, 146 warnings.

The HTML pages are in build\html.
```

**Comment by henriDierkes** [comment]
Solved Spectral Library and Getting Started Warnings

**Comment by janzandr** [comment]
@carlstadie worked on that so far. I will take over now.

**Comment by jakimowb** [comment]
resolves by #87
